scss/components/quote-card/_quote-card.scss

```diff
@@ -33,33 +33,35 @@
     letter-spacing: .5px;
     line-height: 1.3em;
 
-  &:first-of-type {
-    @include grid-media($media-sm-max) {
-      margin-bottom: 0;
+    &:first-of-type {
+      @include grid-media($media-sm-max) {
+        margin-bottom: 0;
+      }
     }
   }
-}
 
-.quote-card__source {
-  position: relative;
 
-  &::before {
-    background-color: color(smoke);
-    content: '';
-    display: block;
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 70px;
-    height: 1px;
-  }
+  .quote-card__source {
+    position: relative;
 
-  strong {
-    color: color(dark-red);
-    display: block;
-    font-style: normal;
-    font-size: em(12px);
-    letter-spacing: .9px;
-    text-transform: uppercase;
+    &::before {
+      background-color: color(smoke);
+      content: '';
+      display: block;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 70px;
+      height: 1px;
+    }
+
+    strong {
+      color: color(dark-red);
+      display: block;
+      font-style: normal;
+      font-size: em(12px);
+      letter-spacing: .9px;
+      text-transform: uppercase;
+    }
   }
 }
```

Comment:
@buttonwillowsix,
I'm not sure what happened here, but it looks like this file is missing a '}' ???

I put it at the end since  `.quote-card__source` is nested within `.quote-card__quote`. Let me know if you see this differently.